src/views/Space/Treasury.vue

```diff
@@ -175,7 +175,12 @@ watchEffect(() => {
             class="mx-4 py-3 border-b flex"
           >
             <div class="flex-auto flex items-center min-w-0">
-              <Stamp :id="asset.contractAddress" type="token" :size="32" />
+              <Stamp
+                :id="`eip155:${treasury.network}:${asset.contractAddress}`"
```

Comment:
@Sekhmet Wouldn't it be simpler to just use network short name instead of eip155 name?